mirascope/tools/web/_parse_url_content.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import annotations
+
+import re
+from typing import ClassVar, TypeVar
+
+import requests
+from bs4 import BeautifulSoup
+from pydantic import Field
+
+from mirascope.tools.base import ConfigurableTool, _ToolConfig
+
+
+class ParseURLConfig(_ToolConfig):
+    """Configuration for URL content parsing"""
+
+    parser: str = Field(
+        "html.parser",
+        description="parser to use for parsing HTML content in BeautifulSoup",
+        examples=["html.parser", "lxml", "lxml-xml", "html5lib"],
+    )
+    timeout: int = Field(
+        default=5,
+        description="Timeout in seconds for URL request",
+    )
+
+    @classmethod
+    def from_env(cls) -> ParseURLConfig:
```

Comment:
same thing here, for these default ones we can remove this method because the base is generic